### ISSUE TITLE
ruby version not specified

### ISSUE THREAD

**Issue by patrick-motard** [pgeraghty/ansible-wrapper-ruby#4]
I'm using ruby version:
```
ruby --version            
ruby 2.6.3p62 (2019-04-16 revision 67580) [x86_64-darwin18]
```
When running `./bin/setup`, `bundler` is the wrong version. Mine is `2.1.4` but the app expects `~>1.1.0`. Assuming that bundler version is tied to ruby version, perhaps we could put a `.ruby-version` file in the repo that specifies what version of ruby should be used, so that we can install it using a ruby version manager such as [rbenv](https://github.com/rbenv/rbenv).

```
 ./bin/setup        
Fetching gem metadata from https://rubygems.org/..........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    bundler (~> 1.10)

  Current Bundler version:
    bundler (2.1.4)
This Gemfile requires a different version of Bundler.
Perhaps you need to update Bundler by running `gem install bundler`?

Could not find gem 'bundler (~> 1.10)' in any of the relevant sources:
  the local ruby installation
```

**Comment by patrick-motard** [comment]
I see [this note](https://github.com/rbenv/rbenv) in the .gemspec file. Don't see anywhere in the project where the expected ruby version is actually specified.

**Comment by patrick-motard** [comment]
I'm new to Ruby. Please excuse my newness ha. I was able to install bundler explicitly:

```
gem install -v "1.10" bundler
```
Then `./bin/setup` worked.
I tried this with two different ruby versions, `2.7.0` and `2.2.6`, so it seems the ruby version is irrelevant.

**Comment by pgeraghty** [comment]
Hi Patrick,

I've relaxed the constraint on Bundler (rake might be next as that's in a similar situation) in the version I've just released. Thanks for your comments - re. `.ruby-version`, but I'm trying to support as wide a range of Rubies as possible (the CI matrix actually goes all the way back to Ruby 2.1, as ridiculous as that might sound).

**Comment by patrick-motard** [comment]
Thanks for responding to quickly! I'd noticed you'd added new versions to ansible-ruby-docker. Wondered if that was related.  I see you have addressed #3  as well. I'm going to test that out now, as well as a more recent version of bundler.
